kernel/safe_typing.mli

```diff
@@ -88,6 +88,10 @@ val export_private_constants :
   private_constants ->
   exported_private_constant list safe_transformer
 
+val export_existing_private_constants :
```

Comment:
Can we document this function?

Reply:
sure, the old entry point was tolerating the two cases dynamically, now you have two entries, one if the effects are in the env, one if they are not.